pkg/generic/generic.go

```diff
@@ -0,0 +1,94 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package generic provides functions and steps implementations not specifically related to Kubernetes nor AWS.
+package generic
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"text/template"
+
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+)
+
+type TemplateArgument struct {
+	Key       string
+	Env_Var   string
+	Fallback  string
```

Comment:
Is `Fallback` same  as  `Default`?

Reply:
This is the intended behavior:
https://github.com/keikoproj/kubedog/blob/51da8e4eb536881aeb4df3b21d13a9f12023595c/pkg/generic/generic.go#L35-L37
I can go with `Default` if you think that word defines it better. Let me know

Reply:
`Default` would be more in convention